tools/packaging/kernel/build-kernel.sh

```diff
@@ -497,7 +497,12 @@ main() {
 	# If not kernel version take it from versions.yaml
 	if [ -z "$kernel_version" ]; then
 		if [[ ${build_type} == "experimental" ]]; then
-			kernel_version=$(get_from_kata_deps "assets.kernel-experimental.tag")
+			if [[ ${arch_target} == "aarch64" ]]; then
+				build_type="arm-experimental"
```

Comment:
Since this isn't just "experimental", this makes it an "architecture specific experimental experimental kernel" right? This could get confusing :smile:

Reply:
Yeah, this is arch-specific experimental kernel. Should we add a "arm-experimental" opt to "build-kernel.sh" to eliminate this confusion?

Reply:
It might be better if we added a new generic option that would only be used if `build_type=experimental"`. That would be a more generic solution and might be clearer. But I'm not sure what that new option would be called ("architecture specific experimental mode"?) But if we can find a name, if that option was specified, the script would use the version defined by `assets.${arch}-kernel-experimental.version` in the versions db.

Reply:
What about adding a new option "-E" for build_kernel.sh and the corresponding build_type would be "arch-experimental". Also add "assets.${arch}-kernel-experimental.version" to versions db. Thus, both of "-a" and "-E" in build_kernel.sh specify the "${arch}-kernel-experimental.version".

Reply:
@jongwu - could you tal at https://github.com/kata-containers/ci/pull/406 as this relates to experimental kernels too.

Reply:
I'd be fine with `arch-experimental`, but it would be good to come to a consensus here.

Reply:
thanks @Jakob-Naucke -, any idea? @devimc @jodh-intel